docs/pages/release_notes.rst

```diff
@@ -5,6 +5,46 @@ Release notes
 
 Whats has been changed, and how to update between versions.
 
-******
-v1.0.0
-******
+***************
+Almond (v1.0.0)
+***************
+
+Release date: 2022-04-01
+
+This is the initial release of the Nuts node reference implementation.
+It implements RFC001 - RFC017 specified by the `Nuts specification <https://nuts-foundation.gitbook.io>`_.
```

Comment:
RFC017 is network v2. This isn't part of nuts-node v1 right?

Reply:
true